browser/lowbrow/menu.js

```diff
@@ -0,0 +1,66 @@
+const modifier = if (process.platform == "darwin") {
+  return "Command";
+} else {
+  return "Control";
+}
+
+const seperator = {
+  type: "seperator"
+}
+
+var shortcut = function() {
+  retrun Array.prototype.slice.call(arguments).join("+")
+}
+
+var menuItem = function(label, accelerator, selector, submenu, click) {
+  return {
+    label: label,
+    accelerator: accelerator,
+    selector: selector,
+    submenu: submenu,
+    click: click
+  };
+}
+
+var menuGroup = function(label, items) {
+  return menuItem(label, null, null, items)
```

Comment:
I hate uses of null as parameters.  If I'm not a heavy user of an API, it's not clear why you are passing null.  To be more clear, how about creating constants like these:
```Javascript
const NO_ACCELERATOR = null;
const NO_SELECTOR = null;
```

And then change this line, and lines like it below, to:
```Javascript
  return menuItem(label, NO_ACCELERATOR, NO_SEPARATOR, items);
```

Seems clearer to me.